include/hal/library/responder/csrlib.h

```diff
@@ -48,6 +48,59 @@ extern bool libspdm_gen_csr(uint32_t base_hash_algo, uint32_t base_asym_algo, bo
                             uint8_t *opaque_data, uint16_t opaque_data_length,
                             size_t *csr_len, uint8_t *csr_pointer,
                             bool is_device_cert_model);
+
+/**
+ * Gen CSR, which is uesd for SPDM 1.3
+ *
+ *   If the device need reset to gen csr, the all case is in the table.
+ *   | Overwrite | Req CSRTtrackingTag | Pending CSR | Reset |             Res Action       |
+ *   |-----------|---------------------|-------------|-------|------------------------------|
+ *   |    No     |          0          |      No     |   -   |          ResetRequired       |
+ *   |    No     |          0          |      Yes    |   -   |     ResetRequired or Busy    |
+ *   |    No     |        Non-0        |   No Match  |   -   |           Unexpected         |
+ *   |    No     |        Non-0        |     Match   | Before|             Busy             |
+ *   |    No     |        Non-0        |     Match   | After |             CSR              |
+ *   |    Yes    |          0          |      No     |   -   |          ResetRequired       |
+ *   |    Yes    |          0          |      Yes    |   -   |          ResetRequired       |
+ *   |    Yes    |        Non-0        |      -      |   -   |             Invalid          |
+ *
+ * @param[in]      base_hash_algo        Indicates the hash algorithm.
+ * @param[in]      base_asym_algo        Indicates the signing algorithm.
+ * @param[in]      need_reset            If true, then device needs to be reset to complete the CSR.
```

Comment:
This is [in/out]. 

On input it gives the value of `CERT_INSTALL_RESET_CAP`.
On output it specifies whether the device needs to be reset to complete the CSR or not.

Reply:
Thanks. I have clear the comment and updated all the [in, out] parameters.